path_role.go

```diff
@@ -10,12 +10,13 @@ import (
 
 	"github.com/hashicorp/vault/sdk/framework"
 	"github.com/hashicorp/vault/sdk/helper/policyutil"
+	"github.com/hashicorp/vault/sdk/helper/tokenutil"
 	"github.com/hashicorp/vault/sdk/logical"
 )
 
 // pathsRole returns the path configurations for the CRUD operations on roles
 func pathsRole(b *azureAuthBackend) []*framework.Path {
-	return []*framework.Path{
+	p := []*framework.Path{
 		&framework.Path{
```

Comment:
While we're here, consider removing the reduntant `&framework.Path`s and `&framework.FieldSchema`s.